user-service/src/main/java/hcmus/alumni/userservice/repository/UserRepository.java

```diff
@@ -54,7 +54,7 @@ int setDataFirstVerifyAlumni(@Param("userId") String userId, @Param("fullName")
 	String findAvatarUrlByUserId(@Param("userId") String userId);
 
 	@Query("SELECT DISTINCT u " + "FROM UserModel u " + "LEFT JOIN u.roles r "
-			+ "WHERE (:fullName IS NULL OR u.fullName LIKE %:fullName%) "
+			+ "WHERE (:fullName IS NULL OR u.fullName LIKE :fullName%) "
```

Comment:
```suggestion
			+ "WHERE (:fullName IS NULL OR u.fullName LIKE %:fullName%) "
```
`%` ở 2 đầu để query có thể nằm ở bất kì vị trí nào thay ví chỉ ở đầu. mấy cái search bên service khác đều theo cách này